code/unit_tests/map_tests.dm

```diff
@@ -0,0 +1,99 @@
+/datum/unit_test/apc_area_test
+	name = "MAP: Area Test APC / Scrubbers / Vents Z level 1"
+
+/datum/unit_test/apc_area_test/start_test()
+	var/list/bad_areas = list()
+	var/area_test_count = 0
+	var/list/exempt_areas = typesof(/area/space,
+					/area/syndicate_station,
+					/area/skipjack_station,
+					/area/solar,
+					/area/shuttle,
+					/area/holodeck,
+					/area/supply/station,
+					/area/mine,
+					/area/vacant/vacant_shop
+					)
+
+	var/list/exempt_from_atmos = typesof(/area/maintenance,
+						/area/storage,
+						/area/engineering/atmos/storage,
+						/area/rnd/test_area,
+						/area/construction,
+						/area/server,
+						/area/mine,
+						/area/vacant/vacant_shop,
+						/area/rnd/research_storage, // This should probably be fixed,
+						/area/security/riot_control // This should probably be fixed,
+						)
+
+	var/list/exempt_from_apc = typesof(/area/construction,
+						/area/medical/genetics,
+						/area/mine,
+						/area/vacant/vacant_shop
+						)
+
+	for(var/area/A in world)
```

Comment:
I'd like to see this split up into different variables, to specifically state if the area is lacking an APC, scrubber or vent.